birdhouse/default.env

```diff
@@ -80,6 +80,24 @@ export SERVER_RELEASE_NOTES_URL='${__DEFAULT__SERVER_RELEASE_NOTES_URL}'
 export SERVER_SUPPORT_URL='${__DEFAULT__SERVER_SUPPORT_URL}'
 export SERVER_LICENSE_URL='${__DEFAULT__SERVER_LICENSE_URL}'
 
+# Defaults for required variables recommended for override for security reasons.
+# Those will not be set explicitly as defaults to ensure they are overridden explicitly by the instance.
+# These values would be detected only if the instance was configured using a copy of 'env.local.example'.
+export __DEFAULT__MAGPIE_SECRET=itzaseekrit
+#export __DEFAULT__MAGPIE_ADMIN_USERNAME=admin
```

Comment:
Is there a reason why these are here but commented? 

Using the default usernames is not recommended either from a security standpoint but is a much lower risk than using the default passwords. 

I would suggest either un-commenting these lines or removing them.

Reply:
I agree, but I found some places where the `admin` is hardcoded.
An example is here: https://github.com/bird-house/birdhouse-deploy/blob/10128e58d51ca3e4b9e4022972944bfdd1f87556/birdhouse/env.local.example#L27

I am not sure if there are other places that had the same assumption. I am not sure also for the user service's usernames other than Magpie if they have similar assumptions in their configurations. If an emitted warning ends up breaking deployments for sure, I prefer to avoid emitting it altogether. However, if we are fine with generating the warning anyway and gradually find where it breaks, I'm fine with uncommenting them as well. This is why I left them there.

Reply:
I see. The fact that they're hardcoded at all is a problem though. Because even without the changes you've made here, we could still override `MAGPIE_ADMIN_USERNAME` and then the `JUPYTERHUB_ADMIN_USERS` would no longer match.

Why don't we make an issue for this and we'll fix it in a later PR

Reply:
Yes. I'm fine with doing another pass in some other PR. I will leave the default usernames commented as reminder.